include/sys/vdev_raidz_impl.h

```diff
@@ -141,6 +141,12 @@ extern const raidz_impl_ops_t vdev_raidz_ssse3_impl;
 #if defined(__x86_64) && defined(HAVE_AVX2)	/* only x86_64 for now */
 extern const raidz_impl_ops_t vdev_raidz_avx2_impl;
 #endif
+#if defined(__x86_64) && defined(HAVE_AVX512F)	/* only x86_64 for now */
```

Comment:
Actually, they are - to my surprise :-)

From the "Intel® 64 and IA-32 Architectures Software Developer’s Manual" chapter 15 "Programming with Intel® AVX-512" they are. Although (15.1.2), "The number of available vector registers in 32-bit mode is still 8."